warehouse/errors.go

```diff
@@ -9,11 +9,7 @@ import (
 	"github.com/rudderlabs/rudder-server/warehouse/internal/model"
 )
 
-var (
-	ErrIncompatibleSchemaConversion = errors.New("incompatible schema conversion")
-	ErrSchemaConversionNotSupported = errors.New("schema conversion not supported")
-	ErrCancellingStatement          = errors.New("[error] pq: canceling statement due to user request")
-)
+var ErrCancellingStatement = errors.New("[error] pq: canceling statement due to user request")
```

Comment:
I cannot find who is returning this error. Can you help me out here? @achettyiitr 

The only place where I see that this error is being used is this one:

```go
		uploadJobsToProcess, err := wh.getUploadsToProcess(ctx, availableWorkers, inProgressNamespaces)

		switch err {
		case nil:
		case context.Canceled, context.DeadlineExceeded, ErrCancellingStatement:
			break loop
```

Reply:
I think we added this to avoid panic. https://github.com/rudderlabs/rudder-server/pull/3105

Reply:
is it ok if I try to take this as a separate task? 
https://www.notion.so/rudderstacks/GetUploadToProcess-error-handling-04fe991f6a90426195f8a9db78fd6e08?pvs=4

Reply:
Right.